README.md

```diff
@@ -47,7 +47,11 @@ Head over to `http://127.0.0.1:8000/docs`
 
 1. `docker-compose up`
 1. Port `80` is mapped to `9000` in the docker compose file so go to `http://127.0.0.1:9000/docs`
-1. Fetch a variant with `curl -X GET "http://127.0.0.1:9000/variants/1_880086_T_C" -H  "accept: application/json"`
```

Comment:
Any particular reason to kill the accept: json? Either is fine in my book..

Reply:
Perhaps it works without?